lecture-notes/3-bld.md

```diff
@@ -1034,26 +1056,26 @@ $ objdump -M intel --disassemble=main main.o
 $
 ```
 
-In particular, it's important to note that the address for the function
-`random_number` is just `0`. You can see this by looking at the `call`
+In particular, it's important to note that the offset for the function
+`random_number` is all `0`s. You can see this by looking at the `call`
 instruction (the x86 opcode for this particular kind of call is `e8` and can be
-found at offset `7`).
+found at offset `7`). The `e8` call opcode is a *relative call*, meaning that
+it takes an offset as its argument. In this case, it's `0xc` because `e8`
```

Comment:
Technically this isn't the argument to the *opcode*. That's zero. `0xc` is the argument to the *mnemonic*, which is subtly different.